tests/cross_functional/system_test/test_nfs_export_in_out_cluster.py

```
@@ -0,0 +1,2001 @@
+        log.info("Step 2: Creating snapshot of NFS PVC")
+
+        snapshot_name = f"{pvc_name}-snapshot"
+        snap_yaml = constants.CSI_CEPHFS_SNAPSHOT_YAML
+
+        # Get the NFS snapshot class (not CephFS)
+        # NFS has its own snapshot class: ocs-storagecluster-nfsplugin-snapclass
+        nfs_snapshotclass_name = "ocs-storagecluster-nfsplugin-snapclass"
+
+        log.info(
+            f"Creating snapshot: {snapshot_name} from NFS PVC: {pvc_name} using "
+            f"snapshot class: {nfs_snapshotclass_name}"
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)